components/LibraryItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import Divider from "components/Divider";
+import LzLink from "components/UI/link";
+import ArrowRightIcon from "public/icons/arrow-right-icon";
+
+export interface ItemProp {
+  type: string;
+  title: string;
+  desc: string;
+  link: string;
+  linkText?: string;
+}
+
+export interface LibraryItemProps {
+  items: ItemProp[]
+}
+
+const Item = (props: ItemProp) => {
```

Comment:
I don't think there's any need to separate the list from the map. Since the lines are not much, you can ask well just loop the whole content.